### ISSUE TITLE
Device Disappears

### ISSUE THREAD

**Issue by dscottbuch** [webdeck/homebridge-indigo#24]
I've been working on getting a stable configuration for Indigo and Homebridge and I'm almost there.   I've move the homekit interface for the majority of devices to an Insteon Hub Pro so they are now stable.  My X-10 device are also now stable through Indigo/Hombridge but I have an instead I/O Linc set up as a Garage opener/sensor which works fine but after about 20 secs always disappears from homekit.  If I restart Homebridge it comes back and works but then just poof.  Any ideas?

It is a virtual device with an action group for up and and action group for down and a variable for state (open/closed).

Thanks,
Scott

**Comment by dscottbuch** [comment]
Things seems to have stabilized, although I'm not sure why.  I did make one change to the virtual device.  For some reason I had specified both an On action and and Off action AND a toggle action.   The only change I made was that I unchecked the Toggle Action choice on the virtual device definition.  I so relieved that It's now stable that I have tried turning it back on to see the impact.  Superstitious I guess :)
